<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_Processing_Widget</class>
 <widget class="QWidget" name="Image_Processing_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="lin_trans_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Linear Transform</string>
   </property>
   <widget class="QLabel" name="alpha_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha (Contrast):</string>
    </property>
   </widget>
   <widget class="QLabel" name="beta_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Beta (Brightness):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="alpha_dspinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="beta_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="contrast_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sharpen_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Sharpen Image</string>
   </property>
   <widget class="QDoubleSpinBox" name="sharpen_spinbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sharpen_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="clahe_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>CLAHE</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Limit:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="clahe_clip_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="clahe_grid_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="clahe_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="bilateral_filter_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Bilateral Filter</string>
   </property>
   <widget class="QCheckBox" name="bilateral_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bilateral_d_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spatial Sigma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Sigma</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bilateral_spatial_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bilateral_color_spinbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
